test/runtime/net_policies.go

```diff
@@ -35,14 +32,11 @@ const (
 	http6PathRewrite  = "http6_path_rewrite"
 
 	// Policy files
-	policyJSON                      = "policy.json"
-	multL7PoliciesJSON              = "Policies-l7-multiple.json"
-	policiesL7JSON                  = "Policies-l7-simple.json"
-	imposePoliciesL7JSON            = "Policies-l7-impose.json"
-	policiesL3JSON                  = "Policies-l3-policy.json"
-	policiesL4Json                  = "Policies-l4-policy.json"
-	policiesL3DependentL7EgressJSON = "Policies-l3-dependent-l7-egress.json"
-	policiesReservedInitJSON        = "Policies-reserved-init.json"
```

Comment:
I guess the removed policy files could be removed from `test/manifests` as well, now that they're unused.